package.json

```diff
@@ -24,18 +24,16 @@
   "homepage": "https://github.com/resin-io/resin-docker-build#readme",
   "dependencies": {
     "@types/bluebird": "^3.0.37",
-    "@types/dockerode": "^2.3.1",
-    "@types/event-stream": "^3.3.30",
+    "@types/dockerode": "2.5.5",
```

Comment:
Aligning this with `resin-builder`:
https://github.com/balena-io/resin-builder/blob/master/package.json#L20
The absence of the caret (`^`) is intentional, see Page's pull request:
https://github.com/balena-io/resin-builder/commit/d09e63603b967b57470e93e9e7f76f4d6eaeaf20